File: support_tools/swigwin-3.0.12/Examples/test-suite/go/namespace_typemap_runme.go
package main

import . "./namespace_typemap"

func main() {
	if Stest1("hello") != "hello" {
		panic(0)
	}

	if Stest2("hello") != "hello" {
		panic(0)
	}

	if Stest3("hello") != "hello" {
		panic(0)
	}

	if Stest4("hello") != "hello" {
		panic(0)
	}

	if Stest5("hello") != "hello" {
		panic(0)
	}

	if Stest6("hello") != "hello" {
		panic(0)
	}

	if Stest7("hello") != "hello" {
		panic(0)
	}

	if Stest8("hello") != "hello" {
		panic(0)
	}

	if Stest9("hello") != "hello" {
		panic(0)
	}

	if Stest10("hello") != "hello" {
		panic(0)
	}

	if Stest11("hello") != "hello" {
		panic(0)
	}

	if Stest12("hello") != "hello" {
		panic(0)
	}

	c := complex(2, 3)
	r := real(c)

	if Ctest1(c) != r {
		println(Ctest1(c))
		panic(Ctest1(c))
	}

	if Ctest2(c) != r {
		panic(0)
	}

	if Ctest3(c) != r {
		panic(0)
	}

	if Ctest4(c) != r {
		panic(0)
	}

	if Ctest5(c) != r {
		panic(0)
	}

	if Ctest6(c) != r {
		panic(0)
	}

	if Ctest7(c) != r {
		panic(0)
	}

	if Ctest8(c) != r {
		panic(0)
	}

	if Ctest9(c) != r {
		panic(0)
	}

	if Ctest10(c) != r {
		panic(0)
	}

	if Ctest11(c) != r {
		panic(0)
	}

	if Ctest12(c) != r {
		panic(0)
	}

	ok := false
	func() {
		defer func() {
			ok = recover() != nil
		}()
		Ttest1(-14)
	}()
	if !ok {
		panic(0)
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/namespace_typemap_runme.py
from namespace_typemap import *

if stest1("hello") != "hello":
    raise RuntimeError

if stest2("hello") != "hello":
    raise RuntimeError

if stest3("hello") != "hello":
    raise RuntimeError

if stest4("hello") != "hello":
    raise RuntimeError

if stest5("hello") != "hello":
    raise RuntimeError

if stest6("hello") != "hello":
    raise RuntimeError

if stest7("hello") != "hello":
    raise RuntimeError

if stest8("hello") != "hello":
    raise RuntimeError

if stest9("hello") != "hello":
    raise RuntimeError

if stest10("hello") != "hello":
    raise RuntimeError

if stest11("hello") != "hello":
    raise RuntimeError

if stest12("hello") != "hello":
    raise RuntimeError

c = complex(2, 3)
r = c.real

if ctest1(c) != r:
    raise RuntimeError

if ctest2(c) != r:
    raise RuntimeError

if ctest3(c) != r:
    raise RuntimeError

if ctest4(c) != r:
    raise RuntimeError

if ctest5(c) != r:
    raise RuntimeError

if ctest6(c) != r:
    raise RuntimeError

if ctest7(c) != r:
    raise RuntimeError

if ctest8(c) != r:
    raise RuntimeError

if ctest9(c) != r:
    raise RuntimeError

if ctest10(c) != r:
    raise RuntimeError

if ctest11(c) != r:
    raise RuntimeError

if ctest12(c) != r:
    raise RuntimeError

try:
    ttest1(-14)
    raise RuntimeError
except ValueError:
    pass
